### PR TITLE
Narrow the scope of the Windows Android arm hack

### PR DESCRIPTION
**Pull request by zanderso**
Engine side of https://github.com/flutter/buildroot/pull/645

### CODE REVIEW

**Comment by zanderso** [comment]
Presubmits are angry at the python script formatting. I'll fix it before landing.

File: DEPS
Comment (godofredoc) [inline DEPS:121]: Will this be updated after https://github.com/flutter/buildroot/pull/645 is landed?

**Review by godofredoc — APPROVED** [review]
LGTM after updating the DEPS file with https://github.com/flutter/buildroot/pull/645

File: tools/gn
Comment (cbracken) [inline tools/gn:330]:
```suggestion
  # Dart tools such as gen_snapshot that are built for the host will correctly
```

**Review by cbracken — APPROVED** [review]
LGTM modulo switching back to flutter/buildroot from zanderso/buildroot as noted by @godofredoc.

File: DEPS
Comment (zanderso) [reply DEPS]: Done.

File: tools/gn
Comment (zanderso) [reply tools/gn]: Done.

**Pull request merged**
